i-VRESSE/bartended-haddock3
# List report.html files on job result page

Refs #17

Once job has completed the job result page looks like
![localhost_3000_jobs_1 (1)](https://user-images.githubusercontent.com/1713488/230418652-82d0e5e4-aacd-46d8-8c19-109c25236218.png)
With links to report.html for each caprieval module analysis.

To test:
1. Setup bartender as instructed at https://github.com/i-VRESSE/bartended-haddock3#bartender-web-service-configuration
2. Create user who has super powers and has guru expertise level, see https://github.com/i-VRESSE/bartender/blob/main/docs/user_management.md#super-user
3. Start with `npm install` and `npm run dev`
4. Create zip from https://github.com/haddocking/haddock3/blob/main/examples/docking-antibody-antigen/docking-antibody-antigen-ranairCDR-clt-test.cfg with `cp docking-antibody-antigen-ranairCDR-clt-test.cfg workflow.cfg` and `zip -r docking-antibody-antigen-ranairCDR-clt-test.zip data workflow.cfg`
5. Either upload zip on http://localhost:3000/builder and submit
6. Goto http://localhost:3000/jobs
7. Wait for job to complete, reload page until you see state is ok or error.
8. Page should look similar to screenshot above.
9. Click on `13_caprieval_analysis` link to see analyis report.
